orion-viewer/src/main/java/universe/constellation/orion/viewer/FileChooser.java

```diff
@@ -46,14 +46,30 @@
 
     private FilenameFilter filter;
 
+    private static Set<String> extensions = new HashSet<>(
+            Arrays.asList("cbz", "djvu", "djv", "pdf", "oxps", "tif", "tiff", "xps"));
+
+
     public static FilenameFilter DEFAULT_FILTER = new FilenameFilter() {
-        public boolean accept(File dir, String filename) {
-        if (new File(dir, filename).isDirectory()) {
-            return true;
+        private String getExtension(String filename) {
+            int index = filename.lastIndexOf(".");
+            if(index == -1) {
+                return "";
+            } else {
+                return filename.substring(index+1);
+            }
         }
-        String name = filename.toLowerCase();
-        return name.endsWith(".pdf") || name.endsWith(".djvu") || name.endsWith(".djv") || name.endsWith(".xps") || name.endsWith(".oxps") || name.endsWith(".cbz")
-                || name.endsWith(".tiff") || name.endsWith(".tif") /*|| name.endsWith(".png") || name.endsWith(".jpeg") || name.endsWith(".jpg")*/;
+
+        public boolean accept(File dir, String filename) {
+            if (new File(dir, filename).isDirectory()) {
+                return true;
+            }
+            if (filename.startsWith("._") || filename.equals(".DS_Store")) {
```

Comment:
Why do you need to filter DS_STORE files? They would be filtered by supported extensions below.